app/src/main/java/edu/uc/coffeens/stocktracker/services/RetrofitClientInstance.kt

```diff
@@ -8,15 +8,15 @@ import retrofit2.converter.gson.GsonConverterFactory
  */
 object RetrofitClientInstance {
 
-    private var retrofit: Retrofit? = null;
-    private val BASE_URL = "https://gist.githubusercontent.com/"
+    private var retrofit: Retrofit? = null
+    private const val BASE_URL = "https://gist.githubusercontent.com/"
```

Comment:
redundant semicolon.
because the URL will never change, the val can be a constant as well.